h2o-py/tests/testdir_apis/H2O_Module/pyunit_h2ono_progress.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import print_function
+import sys
+sys.path.insert(1,"../../../")
+from tests import pyunit_utils
+import h2o
+from h2o.estimators.glm import H2OGeneralizedLinearEstimator
+
+
+def h2ono_progress():
+    """
+    Python API test: h2o.no_progress()
+
+    Command is verified by eyeballing the pyunit test output file and make sure the no progress bars are there.
+    Here, we will assume the command runs well if there is no error message.
```

Comment:
This will test absence of the progress bar:
```
import sys
from io import StringIO
sys.stdout = s = StringIO()
h2o.no_progress()
# ...
# ... (h2o command that normally displays a progress bar)
# ...
assert s.getvalue() == "", "Nothing should have been printed, instead got " + s.getvalue()
sys.stdout = sys.__stdout__  # restore old stdout
```